client/client.go

```diff
@@ -2,14 +2,75 @@ package client
 
 import (
 	"context"
+	"fmt"
+	"strings"
+	"time"
 
 	"github.com/mhemmings/triage/issues"
 )
 
+// IssueFilters contains the possible filters to use when searching for issues.
+type IssueFilters struct {
+	Labels  Labels
+	ShowAll bool
+	Since   Since
+}
+
+// Labels is a slice of strings that implements the gnuflag.Value interface.
+type Labels []string
+
+func (l Labels) String() string {
+	return strings.Join(l, ",")
+}
+
+func (l *Labels) Set(str string) error {
+	*l = strings.Split(str, ",")
+	return nil
+}
+
+// Since is a time.Duration that implements the gnuflag.Value interface.
+type Since time.Duration
+
+func (s Since) String() string {
+	dur := time.Duration(s)
+	if dur == 0 {
+		return ""
+	}
+	return dur.String()
+}
+
+// Set takes a string and assigns the value to the given Since. It supports
+// the time formats from time.ParseDuration, as well as 'd' for day, and
+// 'w' for week.
+func (s *Since) Set(str string) error {
+	var v float64
+	var d time.Duration
+	var err error
+
+	switch {
+	case strings.HasSuffix(str, "d"):
```

Comment:
Hmm, I wonder if this might be better not just as a multiple of 24h, but as an actual date.
I'd perhaps switch the sense of Since so that it stores an absolute time rather than a duration.

Then use `Time.AddDate` for days and weeks, but time.Add for other intervals.
That means it could work correctly with months and years too if you wanted.